cassandane/Cassandane/Cyrus/List.pm

```diff
@@ -2329,4 +2329,55 @@ sub test_list_return_subscribed_trash_nochildren
     });
 }
 
+sub test_scan
+    :min_version_3_9
+{
+    my ($self) = @_;
+
+    my $store = $self->{store};
+    my $imaptalk = $store->get_client();
+
+    $self->setup_mailbox_structure($imaptalk, [
+        [ 'create' => [qw( INBOX.a INBOX.b INBOX.c )] ],
+    ]);
+
+    xlog $self, "listing...";
+    my $res = $imaptalk->list("", "*");
+warn Dumper($res);
```

Comment:
can you remove this warning though please :)

Reply:
done